tests/tests/src/pattern_match_json.mjs

```diff
@@ -25,7 +25,11 @@ function decodeGroup(group) {
 }
 
 function decodeNull(x) {
-  let tmp = x.field;
+  let match = x.field;
```

Comment:
This has changed because dicts always add optional fields, and in this case it contains a json value, and one of the possible json values is `undefined`.